newlib/testsuite/newlib.iconv/iconvjp.c

```diff
@@ -968,6 +968,7 @@ int main(void)
 #else /* #if defined(_ICONV_FROM_ENCODING_UTF_8) || ... */
 int main(int argc, char **argv)
```

Comment:
You can also change the definition of main to be just `int main (void)` so that you don't have the arguments at all.

Reply:
Sorry, bad case of tunnel vision, that's obviously what I should have done. Fixed.